src/scss/atom/_button.scss

```diff
@@ -0,0 +1,61 @@
+@import "../helper/variables";
+@import "../helper/mixin";
+
+.button {
+    @include invisible-button;
+    display: inline-block;
+    font-size: 1.4rem;
```

Comment:
Do we need to set the font-size on the button or is this just the default font-size?